go/cmd/vtctldclient/command/root.go

```diff
@@ -26,6 +26,11 @@ import (
 
 	"github.com/spf13/cobra"
 
+	// These imports ensure init()s within them get called and they register their commands/subcommands.
+	vreplcommon "vitess.io/vitess/go/cmd/vtctldclient/command/vreplication/common"
```

Comment:
regarding the cobra stuff, i kinda don't understand why we landed on this structure vs putting all the workflow stuff in `workflows.go`, movetables stuff in `movetables.go`, and so on

Reply:
This structure was a result my efforts to understand movetables for porting over `Reshard`. The pros I see, it
* keeps related code to subcommands separate
* felt more intuitive especially with the common subcommands in MoveTables and Reshard
* made it easier to extract common code into utility helpers
* will also make it easier to port over some of the other commands like `Materialize`, `Migrate` and `CreateLookupIndex` which share a lot of components from `MoveTables`